### ISSUE TITLE
MRCNN accuracy

### ISSUE THREAD

**Issue by a-maci** [pytorch/vision#1140]
I ran `maskrcnn` on `coco` dataset with the following command line:
`python -m torch.distributed.launch --nproc_per_node=4 --use_env ../references/detection/train.py --data-path=/coco`

(everything else are default parameters) and I get the following output:
```
IoU metric: bbox
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.36683
IoU metric: segm
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.33667
```

Seems that the numbers are lower than what the torchvision page reports (`37.9 (bbox) | 34.6 (mAP)`). What am I doing wrong?

**Comment by fmassa** [comment]
I trained for more epochs (roughtly the 2x schedule).

Here is the command I used

```
python -m torch.distributed.launch --nproc_per_node=8 --use_env train.py --dataset coco --model maskrcnn_resnet50_fpn --epochs 26 --lr-steps 16 22 --aspect-ratio-group-factor 3
```
